tests/cnf/core/network/metallb/internal/frr/frr.go

```diff
@@ -137,6 +137,41 @@ type (
 
 	// BGPNeighborGRStatus is a map of GRStatus per peer.
 	BGPNeighborGRStatus map[string]GRStatus
+
+	// InterfaceDetails retrieved from FRR pod interface.
+	InterfaceDetails struct {
+		AdministrativeStatus string `json:"administrativeStatus"`
+		OperationalStatus    string `json:"operationalStatus"`
+		LinkDetection        bool   `json:"linkDetection"`
```

Comment:
I am in favor of having smaller struct with only the fields you need. Adding new field if needed should be easy

Reply:
removed unused vars